keyboards/gh60/satan/keymaps/gipsy-king/keymap.c

```diff
@@ -0,0 +1,159 @@
+#include QMK_KEYBOARD_H
+#include "rgblight.h"
+
+enum layer_names {
+  _BL,
+  _FL
+};
+
+/**
+ * HHKB style.
+ * Esc on capslock, space-hold is fn.
+ * Fn layer has hjkl arrows, home on backspace, rgb stuff.
+ */
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BL] = LAYOUT_60_ansi(
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
thank you